addons/event/security/ir.model.access.csv

```diff
@@ -26,3 +26,9 @@ access_event_tag,event.tag,model_event_tag,,0,0,0,0
 access_event_tag_registration,event.tag.user,model_event_tag,event.group_event_registration_desk,1,0,0,0
 access_event_tag_user,event.tag.user,model_event_tag,event.group_event_user,1,1,1,0
 access_event_tag_manager,event.tag.manager,model_event_tag,event.group_event_manager,1,1,1,1
+access_event_question_manager,event.question,model_event_question,event.group_event_manager,1,1,1,1
+access_event_question_user,event.question.user,model_event_question,event.group_event_user,1,1,1,1
+access_event_question_answer_employee,event.question.answer,model_event_question_answer,event.group_event_user,1,0,0,0
+access_event_question_answer_registration,event.question.answer.registration,model_event_question_answer,event.group_event_registration_desk,1,1,0,0
```

Comment:
It is not normal that registration desk users have more rights than event users ... hierarchy is registration desk < event user < event manager. I suppose we should give 1,1,1,1 ?